sympy/physics/vector/tests/test_point.py

```diff
@@ -324,3 +326,53 @@ def test_auto_vel_derivative():
     Cm._vel_dict = temp
     Cm.v2pt_theory(Bm, B, C)
     assert Cm.vel(A) == (u1 * B.y + (u1 + u2) * C.y)
+
+def test_auto_point_acc_zero_vel():
+    N = ReferenceFrame('N')
+    O = Point('O')
+    O.set_vel(N, 0)
+    assert O.acc(N) == 0 * N.x
+
+def test_auto_point_acc_compute_vel():
+    t = dynamicsymbols._t
+    q1 = dynamicsymbols('q1')
+    N = ReferenceFrame('N')
+    A = ReferenceFrame('A')
+    A.orient_axis(N, N.z, q1)
+
+    O = Point('O')
+    O.set_vel(N, 0)
+    P = Point('P')
+    P.set_pos(O, A.x)
+    assert P.acc(N) == -q1.diff(t) ** 2 * A.x + q1.diff(t, 2) * A.y
```

Comment:
Should you also check that the vel_dict is populated even though .vel() isn't explicitly called?

Reply:
`vel_dict` is indeed changed because we call `vel` in the `acc` method.